example/package.json

```diff
@@ -20,7 +20,6 @@
     "eslint-plugin-react": "^7.14.3",
     "get-dev-paths": "^0.1.1",
     "metro-config": "^0.56.0",
-    "npm-check-updates": "^3.1.21",
```

Comment:
LOL this line triggered 3.5k additions and 5.4k removals. gotta love npm. 
I'm a bit confused as to why this fixes the dot-prop dependency, though. was it a dependency of `npm-check-updates`?

Reply:
I know haha. Yes, `dot-prop` was a dependency of `npm-check-updates`